src/main/java/io/projectriff/invoker/server/ConsumerFunction.java

```diff
@@ -0,0 +1,22 @@
+package io.projectriff.invoker.server;
+
+import org.springframework.messaging.Message;
+import reactor.core.publisher.Flux;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class ConsumerFunction<T> implements Function<T, Object> {
```

Comment:
Using `Void` as return type could better convey the Consumery aspect of the function

Reply:
That's indeed cleaner from a type perspective. However, that means I cannot return an empty Flux and I would need to update `io.projectriff.invoker.server.GrpcServerAdapter#promoteToArray` to handle a new special case.